Gemfile

```diff
@@ -55,6 +55,8 @@ group :development, :test do
   gem 'selenium-webdriver'
   gem 'solr_wrapper'
   gem 'webmock'
+  # factory_bot_rails for creating fixtures in tests
+  gem 'factory_bot_rails', '~> 6.2.0' # Pinned until https://github.com/thoughtbot/factory_bot_rails/issues/433 is resolved
```

Comment:
That issue was resolved already.  Is there a reason we can't use 6.4?

Reply:
I was copy-pasting from the requests gemfile, so I will update this one to remove the version restriction.